stdlib/public/core/OptionSet.swift

```diff
@@ -374,3 +374,15 @@ extension OptionSet where RawValue: FixedWidthInteger {
     self = Self(rawValue: self.rawValue ^ other.rawValue)
   }
 }
+
+extension OptionSet where RawValue: FixedWidthInteger, Element == Self {
+  @_alwaysEmitIntoClient
```

Comment:
To double check, `@inlineable` isn't required because we have `@_alwaysEmitIntoClient` right?